### DISCUSSION TITLE
New installation (with files from another installation) is stuck on "Waiting for master server"

### DISCUSSION THREAD

**Discussion by ODA-AG** [jhuckaby/Cronicle#425] [Q&A]
We want to keep the Users/API Keys/Schedules from one instance of Cronicle and copy them (conf/data folders) to a new instance. The networks are separate so they won't see each other, but it seems to hang on Waiting for master server...

**Comment by ODA-AG** [comment]
When I try to use the Cronicle Data Importer, it errors with:

```
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/opt/cronicle/import/otherserver.txt'
Emitted 'error' event at:
    at lazyFs.open (internal/fs/streams.js:115:12)
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)

```

**Comment by mikeTWC1984** [comment]
Check some recent "Discussion" topics below, this is the most common problem. I guess you just need to use storage-cli.js export/import tool. The reason for "Waiting for master" is master node name mismatch (keeps old server reference on the new cluster)

**Reply by ODA-AG** [reply]
When I run /opt/cronicle/bin/control.sh setup command, I get "Storage has already been set up. There is no need to run this command again."

Should I delete some folders?

**Reply by mikeTWC1984** [reply]
When you initially set up cronicle, this command should set up data folder. Are you just copy/pasting data folder?

**Reply by ODA-AG** [reply]
I just gave up and set it up clean and manually configured it. So it is working for me now.
